src/core/PluginManager.cpp

```diff
@@ -27,16 +28,14 @@ namespace collabvm::core {
 				vsprintf(&buf[0], mp, va);
 				va_end(va);
 
-				// hope your compiler supports this!
-				using enum plugin::IPluginApi::LogLevel;
```

Comment:
I don't think it would be any different from with the `using enum` statement. Anyways, with no plugins to test (even though the sample plugin exists, the engine isn't just ready to run plugins yet) we can't really experiment with this yet.